wifiphisher/common/opmode.py

```diff
@@ -183,18 +193,34 @@ def set_opmode(self, args, network_manager):
         6) Advanced and Internet w/ 1 vif support AP/Monitor 0x6
           2 cards, 3 interfaces
           i) AP, ii) Extensions, iii) Internet
+        7) Advanced and WPS association 0x7
+          3 cards, 3 interfaces
+          i) AP, ii) Extensions, iii) WPS association
+        8) Advanced and WPS association w/ 1 vif support AP/Monitor 0x8
+          2 cards, 3 interfaces
+          i) AP, ii) Extensions, iii) WPS association
```

Comment:
This needs some thinking. We need to think of a global solution for potential future modules.

The question is: Why can we not use the second interface (for Extensions) and we need a third one? Because the third one acts as a STATION (in Managed mode). Hence, we probably need to make this more generic and instead of "WPS association" mention it as "Extensions (Managed)" as opposed to "Extensions (Monitored).

Future extensions that also need to act as a station in Managed Mode will also fall to this category.

Reply:
I will think about this one

Reply:
If you agree with my thinking, you can just change the documentation above and instead of "WPS association" write "Extensions (Managed)". Also instead of "Extensions" write "Extensions (Monitor)"

Reply:
yeah, @sophron I agree with your thinking. While I can change the documentation we still need the third card for the current PR.

If we just want to use two cards to implement this PR we need to carefully think about the one phy card case, since for one card it will use as both AP and Monitor, but during WPS association the `monitor mode` of this card needs to change to `managed` do scan and connect, this process will try to change the channel which I'm not sure if there will be problem or not.

If you think it is ok to change the documentation, I can change it first, then we may improve this for the future PR.